develop/CICD/Tutorials/CICD_Tutorial_GitHub_Setting_Up_Organization.md

```diff
@@ -0,0 +1,92 @@
+
+Expected duration: 10 minutes.
+
+## Private versus public code
+
+Both SonarCloud and GitHub require paid licenses to fully support private GitHub repositories.
+
+For real-world scenarios, you will likely need at least a paid SonarCloud license to work with private GitHub repositories. A free GitHub version can still be used for private repositories, but this will slightly alter how you work with the product.
+
+## Prerequisites
+
+- [GitHub account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account)
+
+## Overview
+
+- [Step 1: Create a GitHub organization](#step-1-create-a-github-organization)
+- [Step 2: Create a SonarCloud organization and link it with GitHub](#step-2-create-a-sonarcloud-organization-and-link-it-with-github)
+- [Step 3: Set up starter workflows for your organization](#step-3-set-up-starter-workflows-for-your-organization)
+- [Step 4: Adjust permissions for GITHUB_TOKEN](#step-4-adjust-permissions-for-github_token)
+- [Step 5: Optionally add organization-wide secrets](#step-5-optionally-add-organization-wide-secrets)
+
+## Step 1: Create a GitHub Organization
+
+Follow the instructions specified in the [GitHub documentation](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/creating-a-new-organization-from-scratch).
+
+## Step 2: Create a SonarCloud organization and link it with GitHub
+
+Follow the instructions specified in the [SonarCloud documentation](https://docs.sonarsource.com/sonarcloud/getting-started/github/).
+
+## Step 3: Set up starter workflows for your organization
+
+1. Optionally, create a GitHub user specifically for bot activities, e.g. *CICDOrganization*.
+
+   - Use a shared mailbox from your organization if possible. If this is not possible, for example because of 2FA, you can use a personal GitHub account with rights to create repositories.
+
+   - Add the user to your organization and ensure they have sufficient rights to create new repositories.
+
+1. Go to *User Settings* > *Developer Settings* > *Personal Access Tokens* to create a personal access token (classic) for your GitHub user.
+
+   Make sure the token has the following scopes:
+
+   - repo
+   - workflow
+   - admin:org/write:org
+   - admin:org/read:org
+
+   > [!TIP]
+   > For more details, refer to the [GitHub documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).
+
+1. [Fork the *Skyline-Starter-Workflows* repository](https://github.com/SkylineCommunications/Skyline-Starter-Workflows/fork) into your organization.
+
+1. Add a new secret to the fork:
+
+   1. Go to the settings of your fork and select *Secrets and variables* > *Actions*.
+
+   1. Add a new secret called `PAT` and use the previously created personal access token as the value.
+
+1. Go to the *Actions* tab, verify the workflow content, and enable the workflow.
```

Comment:
For these two steps (enable workflow + trigger initial run), I think we might need to add more detail so it's definitely clear how to do this. Maybe you could add screenshots?

Reply:
to avoid duplicate work, would it be ok to reference uid: GitHub_Secrets  specifically ## Adding the key as a secret in the repository

I added that file, because I was noticing having to explain that in several places.

Reply:
That reference would help for the previous step, but not for this one and the next one, because it doesn't explain how to enable a workflow and trigger an initial run.